code/modules/ai/ai_behaviors/xeno/xeno.dm

```diff
@@ -104,6 +104,33 @@
 		change_action(MOVING_TO_ATOM, next_target)
 		return
 
+/*
+/datum/ai_behavior/human/look_for_new_state(atom/next_target)
+	if(human_ai_state_flags & HUMAN_AI_ANY_HEALING)
+		return
```

Comment:
?

Reply:
Uhhh you want to delete this? I think I just pasted it over to compare and forgot